resource/resource.go

```diff
@@ -19,10 +19,17 @@
 // and stylesheets.
 package resource
 
-// When changing files included here, you may add the `-debug` flag, which will
-// avoid embedding any files (but instead passes them through from disk). This
-// avoids having to recompile before testing changes.
-
+// If you add any files to the ui folder, you'll need to generate and make build
```

Comment:
I would mention this specifically in the context with `-debug`: Even though go-bindata will just proxy through, it will *only* proxy those files that were there when you last ran `go generate`.